crates/astria-cli/src/commands/rollup.rs

```diff
@@ -99,9 +103,24 @@ fn update_yaml_value(
 /// * If the config file cannot be created
 /// * If the arguments cannot be serialized to yaml
 /// * If the yaml cannot be written to the file
-pub(crate) fn create_config(args: &ConfigCreateArgs) -> eyre::Result<()> {
+pub(crate) async fn create_config(args: &ConfigCreateArgs) -> eyre::Result<()> {
     // create rollup from args
-    let rollup = Rollup::try_from(args)?;
+    let mut conf = (*args).clone();
```

Comment:
```suggestion
    let mut conf = args.clone();
```
don't think you need the deref?